<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="fit_widget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fit model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_file_widget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filepath_field">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C:\Users\cdami\Documents\LinearRegression\doc.txt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lr_field">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.001</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="epoch_field">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>120</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>421</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="chart_field"/>
   </widget>
   <widget class="QLabel" name="epoch_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Epochs_elapsed: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="eq_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Line equation: y = 0x + 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="loss_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Loss = inf</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
